### ISSUE TITLE
🤔 [QUESTION]

### ISSUE THREAD

**Issue by tomasvillamorgm** [antvis/G2Plot#3641]
HI! When i am trying to use tooltip sometimes i dont need to display information, but i dont now how.

If approve certain condition I don't need to show anything
For example:

![image](https://github.com/antvis/G2Plot/assets/89924297/ca722559-ccbd-40f3-922c-308f19f8009b)

this code show the tooltip as this way
![image](https://github.com/antvis/G2Plot/assets/89924297/131caef4-7d54-49f2-8314-be2fc8dc0dd8)

In conclusion, i want to appear the tooltip but empty, as the example without the blue circle and the points.
* **G2Plot Version**:
* **Platform**:
